### ISSUE TITLE
Electron-helper crashing without info

### ISSUE THREAD

**Issue by cobarx** [chentsulin/electron-react-boilerplate#342]
I'm running into one other issue with nodobjc. I have bit of code I'm trying to execute that works fine in the Electron quick start project but seems to bomb one of the electron helpers with this project.

When it happens I get the following error:
```
DevTools was disconnected from the page.

Once page is reloaded, DevTools will automatically reconnect.
```

Is there any way to setup logging for electron-helper, I don't get anything in the Chrome Debugger or terminal when this happens.

**Comment by amilajack** [comment]
Usually that means that you have some overwhelmingly expensive code that causes electron to crash.

**Comment by cobarx** [comment]
It's fairly simple code. Is there any way to get instrumentation and see what might be eating up all the cycles? The same code works fine in a simpler setup without webpack, etc.

**Comment by amilajack** [comment]
What code did you add that causes that error?

**Comment by cobarx** [comment]
```
const ChipEvent = $.NSObject.extend('ChipEvent');
ChipEvent.addMethod('chipStatusChanged:', 'v@:@', function(self, _cmd, mount) {
  console.log("Chip status changed " + mount('path'));
});
ChipEvent.register();
var event = ChipEvent('alloc')('init');
const chipSDK = $.ChipSDK("alloc")("initWithEvent", event);
```

ChipEvent is basically a nodobjc implementation of a Objective C protocol. chipStatusChanged gets invoked as a callback whenever a you hook up a flash drive. In the ChipSDK, I've got an NSLog immediately before chipStatusChanged gets called and see that get printed.

**Comment by amilajack** [comment]
1. Try doing this in the file separate (say ./app/some-test.js)
2. run it: `node app/some-test.js`
3. Check if it runs correctly

If that work doesnt work, its probably a webpack configuration issue.

**Comment by amilajack** [comment]
Closing due to no response.

**Comment by mohammedgqudah** [comment]
Same issue here 😕 ...
```javascript
const electron = require('electron');
const { app, BrowserWindow} = electron;

app.on('ready', () => {
    new BrowserWindow({});
});
```

**Comment by redstubble** [comment]
This issue has been driving me crazy, I cannot see any expensive code that is taking place. It has happened in multiple places in my code. It makes this whole boilerplate pointless. I am subscribing to a websockets connection and I can't do any debugging. At my wits end on how to resolve this issue.

**Comment by alexeypetrushin** [comment]
Same issue... no expensive code, just following the tutorial and added code to read some file and display it on the page

**Comment by ngr900** [comment]
I was having the same issue, although not with `electron-react-boilerplate`, just an Electron app. In my case the code that caused the crash initialized a few `Workers`. The code inside the `Workers` was not the cause of the problem however, DevTools crashed even if the `Worker` script was literally empty or just contained a `console.log('test')`.

What I found was that the crashes were caused by having `nodeIntegrationInWorker` set to `true`. I had to `require` some libraries inside the `Workers`, but I got around the issue by using `importScript` and manually downloading the library files.
